kubespawner/reflector.py

```diff
@@ -218,9 +218,12 @@ async def _list_and_update(self):
         kwargs = dict(
             label_selector=self.label_selector,
             field_selector=self.field_selector,
+            resource_version=resource_version,
             _request_timeout=self.request_timeout,
```

Comment:
I do think it would be a good idea to change this timeout to a larger value for the watches, even if cached lists are still very expensive. Other k8s components typically relist watches in the range of minutes to never, every 10 seconds is a bit much.
Though probably should be done in a different PR.

Reply:
Yep let's consider that in a different PR